internal/reader/home_chain.go

```diff
@@ -116,7 +117,12 @@ func (r *homeChainPoller) poll() {
 func (r *homeChainPoller) fetchAndSetConfigs(ctx context.Context) error {
 	var chainConfigInfos []ChainConfigInfo
 	err := r.homeChainReader.GetLatestValue(
-		ctx, consts.ContractNameCCIPConfig, consts.MethodNameGetAllChainConfigs, nil, &chainConfigInfos,
+		ctx,
+		consts.ContractNameCCIPConfig,
+		consts.MethodNameGetAllChainConfigs,
+		primitives.Finalized,
```

Comment:
Ditto here, what other options are there?

Reply:
Finalized or Unconfirmed. Why would we want an unfinalized data?

Reply:
Does not need to be finalized, can use Unconfirmed.